docs/data-sources/resource.md

```diff
@@ -38,16 +38,16 @@ data "twingate_resource" "foo" {
 Read-Only:
 
 - `allow_icmp` (Boolean) Whether to allow ICMP (ping) traffic
-- `tcp` (Block List) (see [below for nested schema](#nestedblock--protocols--tcp))
-- `udp` (Block List) (see [below for nested schema](#nestedblock--protocols--udp))
+- `tcp` (Block, Read-only) (see [below for nested schema](#nestedblock--protocols--tcp))
```

Comment:
Why is there a change (`Read-only`) here and not in `allow_icmp` ?

Reply:
it's the same situation as in previous comment